shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -241,7 +241,8 @@
   gesture_recognizers_blocking_policies[idString] = gestureRecognizerBlockingPolicy;
 }
 
-void FlutterPlatformViewsController::SetFrameSize(SkISize frame_size) {
+void FlutterPlatformViewsController::BeginFrame(SkISize frame_size) {
+  composition_order_.clear();
```

Comment:
fyi: Android does this: https://github.com/flutter/engine/blob/master/shell/platform/android/external_view_embedder/external_view_embedder.cc#L265.

Reply:
could we also have a unit test for this?

Reply:
could we make a`Reset()` method, similar to Android?  Why not also clearing `picture_recorders_`?